ndoh_hub/utils.py

```diff
@@ -138,29 +136,11 @@ def get_messageset_schedule_sequence(short_name, weeks):
     # get messageset
     messageset = get_messageset(short_name)
 
-    messageset_id = messageset["id"]
-    schedule_id = messageset["default_schedule"]
-    # get schedule
-    schedule = get_schedule(schedule_id)
-
     # calculate next_sequence_number
-    # get schedule days of week: comma-seperated str e.g. '1,3' for Mon & Wed
-    days_of_week = schedule["day_of_week"]
-    # determine how many times a week messages are sent e.g. 2 for '1,3'
-    msgs_per_week = len(days_of_week.split(','))
-
-    # determine starting message
-    if 'postbirth' in short_name:
-        next_sequence_number = 1  # start baby messages at 1
-    elif 'loss' in short_name:
-        next_sequence_number = 1  # always start loss messages at 1
-    else:
-        next_sequence_number = msgs_per_week * (
-            weeks - settings.PREBIRTH_MIN_WEEKS)
-        if next_sequence_number == 0:
-            next_sequence_number = 1  # next_sequence_number cannot be 0
+    next_sequence_number = 1  # to be expanded with momconnect migration
```

Comment:
Scope confusion has ensued. Waiting on a response from matthew and will open another issue depending on what he says.